### DISCUSSION TITLE
Zim version 0.74.1 released !

### DISCUSSION THREAD

**Discussion by jaap-karssenberg** [zim-desktop-wiki/zim-desktop-wiki#1618] [Announcements]
Dear all,

Zim version 0.74.1 is now available. It is a small bug fix release of the 0.74.0 release of earlier this month.

Also I now added a windows installer to the download folder.

Kind regards,

Jaap

##  0.74.1 - Thu 23 Sep 2021
* Add preference to hide edit bar
* Add separate parameter for wrapped lines linespacing
* Fix behavior of copying images on copy-paste
* Fix export of anchors to make valid HTML tags
* Fix storing of uistate for window panes
* Fix issues for "file:/path" file URIs
* Fix bug in "copy link"
* Make clipboard handling robust for non-local file URI
* Fix issue with escaping in find input
* Fix behavior of TrashDialog on error
* Remove string literal formatting syntax

**Comment by Nick222** [comment]
Thank you :)

Excuse me - I cannot understand:

> Add separate parameter for wrapped lines linespacing

**Comment by jaap-karssenberg** [comment]
The style configuration now has a separate parameter to control the line
spacing within wrapped lines.

Op vr 24 sep. 2021 19:32 schreef Nick222 ***@***.***>:
